source/renderer/app/components/staking/delegation-center/DelegationCenter.js

```diff
@@ -1,33 +1,39 @@
 // @flow
 import React, { Component, Fragment } from 'react';
 import { observer } from 'mobx-react';
-import BigNumber from 'bignumber.js';
 import DelegationCenterHeader from './DelegationCenterHeader';
 import DelegationCenterBody from './DelegationCenterBody';
 import Wallet from '../../../domains/Wallet';
+import type { NextEpoch, TipInfo } from '../../../api/network/types';
 
 type Props = {
-  adaValue: BigNumber,
-  percentage: number,
   wallets: Array<Wallet>,
   onDelegate: Function,
   numberOfStakePools: number,
+  redirectToStakingInfo: Function,
+  networkTip: ?TipInfo,
+  nextEpoch: ?NextEpoch,
 };
 
 @observer
 export default class DelegationCenter extends Component<Props> {
   render() {
     const {
-      adaValue,
-      percentage,
       wallets,
       onDelegate,
       numberOfStakePools,
+      redirectToStakingInfo,
```

Comment:
@DeeJayElly I don't think we need this one. We used this redirect on the Delegation Countdown screen but here on the Delegation Center there should be no redirects!

Reply:
Fixed